curricula/templates/curricula/unit.html

```diff
@@ -278,202 +278,6 @@ <h4>
-                        } else {
-                            $("#publish_pdfs").addClass('btn-warning').text("Failed");
-                            $("#publish_results_pdf").text(JSON.stringify(response, null, 2));
-                            $("#publish_results_pdf").append('\nFailed');
-                            $('#progress_spinner_pdf').removeClass('fa fa-cog fa-spin');
-                        }
-                    }
-                });
-                pdf_client.send();
-                /*
-                $.ajax({
-                    type: "POST",
-                    url: "/publish/",
-                    data: {pk: pk, type: type, pdf: true},
-                    timeout: 9999999
-                }).done(function(response) {
-                    $("#publish_pdfs").addClass('btn-success').text("Success");
-                    $("#publish_results_pdf").text(JSON.stringify(response, undefined, 2));
-                    $('#progress_spinner_pdf').removeClass('fa fa-circle-o-notch fa-spin');
-                }).fail(function(response) {
-                    $("#publish_pdfs").addClass('btn-warning').text("Failed");
-                    $("#publish_results_pdf").text(JSON.stringify(response, undefined, 2));
-                    $('#progress_spinner_pdf').removeClass('fa fa-circle-o-notch fa-spin');
-                });
-                */
-            });
-
-            // Clone button
-            $("#clone_this").click(function () {
-                $('#clone_this').html("Cloning...").removeClass('btn-warning btn-success').addClass('btn-primary');
-                var pk = $(this).attr('data-pk');
-                var type = $(this).attr('data-type');
-                var children = $('#clone_children').prop('checked');
-                $('#clone_spinner').addClass('fa fa-cog fa-spin');
-
-                $.ajax({
-                     type: "POST",
-                     url: "/clone/",
-                     data: {pk: pk, type: type, children: children},
-                     timeout: 9999999
-                }).done(function (response) {
-                    $("#clone_this").addClass('btn-success').text("Success");
-                    $("#clone_results").text(JSON.stringify(response, undefined, 2));
-                    $('#clone_spinner').removeClass('fa fa-cog fa-spin');
-                    if (response.hasOwnProperty('redirect_url')) {
-                        setTimeout(function () {
-                            window.location.href = response.redirect_url;
-                        }, 500);
-                    }
-                }).fail(function (response) {
-                    $("#clone_this").addClass('btn-warning').text("Failed");
-                    $("#clone_results").text(JSON.stringify(response, undefined, 2));
-                    $('#clone_spinner').removeClass('fa fa-cog fa-spin');
-                });
-            });
-        });
-    </script>
+    <script src="{% static "js/admin_menu.js" %}"></script>
```

Comment:
is there a reason why we only sometimes gate this with a `user.is_staff` check? Seems like that should always happen

Reply:
This is gated in `user.is_staff`, it's just hiding above that huge mess of removed lines :)